src/main/perl/Options.pm

```diff
@@ -246,6 +259,39 @@ sub action_showcids
     return SUCCESS;
 }
 
+=item add_actions
+
+Add actions defined in hashref to the supported actions.
+
+When creating a new module derived from EDG::WP4::CCM::Options,
+add methods named "action_<something>", and add then via this method
+to the _actions hashref.
+
+This will create a commandline option "--something", if selected,
+will execute the action_<something> method.
+
+The hashref key is the action name, the value is the help text.
+
+(Returns the _actions hashref for unittesting purposes)
+
+=cut
+
+sub add_actions
+{
+    my ($self, $newactions) = @_;
+
+    while (my ($action, $help) = each %$newactions) {
+        my $method = $self->can("action_$action");
```

Comment:
This variable seems a bit redundant? Is there a reason not to avoid the extra variable and call:
``` if($self->can("action_$action")) {```